internal/provider/resource_consul_cluster_test.go

```diff
@@ -1,3 +1,5 @@
+// This includes tests against both the resource and the corresponding datasource
```

Comment:
thanks for adding these! last thing: can you bump this down to above L33? Same for the others? I don't usually see comments above the package in go so I think this'll be easy to miss.